File: src/main/java/org/jdc/template/log/DebugTree.java
package org.jdc.template.log;

import timber.log.Timber;

public class DebugTree extends Timber.DebugTree {

    @Override
    protected String createStackElementTag(StackTraceElement element) {
        // add line number
        return super.createStackElementTag(element) + ":" + element.getLineNumber();
    }
}

File: src/main/kotlin/org/jdc/template/log/DebugTree.kt
package org.jdc.template.log

import timber.log.Timber

class DebugTree : Timber.DebugTree() {

    override fun createStackElementTag(element: StackTraceElement): String {
        // add line number
        return super.createStackElementTag(element) + ":" + element.lineNumber
    }
}
